ArduPlane/system.cpp

```diff
@@ -120,7 +120,11 @@ void Plane::init_ardupilot()
     // don't initialise aux rc output until after quadplane is setup as
     // that can change initial values of channels
     init_rc_out_aux();
-    
+
+    if (g2.oneshot_mask != 0) {
```

Comment:
```suggestion
    if (g2.oneshot_mask > 0) {
```

Reply:
.... what's this doing in this PR, anyway?